CONTRIBUTING.md

```
@@ -189,8 +189,53 @@
+2. **Push your changes to remote**
+3. **Validate your changes in Development**
+
+    > [!NOTE]
+    >
+    > Please check out the [Validating Changes to the Bot Locally](https://github.com/innovateorange/DiscordBot/blob/main/CONTRIBUTING.md#validating-changes-to-the-bot-locally) section of this document if you need to run and test / validate your changes to the bot locally
+4. **Merge branch into `stage`**
+
+   > [!IMPORTANT]
+   >
+   > Do **NOT** create a pull request. Only proceed with the following commands **after you have validated your changes in the `Development` environment**
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)